### DISCUSSION TITLE
Tekton Trigger and Kubernetes Version Compatibility

### DISCUSSION THREAD

**Discussion by sunyeongchoi** [tektoncd/triggers#1559] [Q&A] [answered]
The `Tekton Pipeline` documentation has information about Kubernetes Versions.

However, I couldn't find it anywhere in the `Tekton Trigger` documentation.

Can I find any documentation related to your Kubernetes version?

**Comment by khrm** [comment]
We don't have any such page atm. I will create this page today.

**Reply by sunyeongchoi** [reply]
thank you so much :)  
Will that be updated in the README.md of the main branch?

**Reply by khrm** [reply]
No. A different page.

**Reply by sunyeongchoi** [reply]
Okay. Thank you :>

**Reply by khrm** [reply]
https://github.com/tektoncd/triggers/pull/1560

**Comment by khrm** [comment, marked as answer]
For v0.23.x, Kubernetes minimum version is 1.24
